.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -42,6 +46,7 @@ in place of the last 2 commands above.
 ---
 
 Still not clear ???
-see example - https://checkstyle.org/report_issue.html#How_to_report_a_bug.3F
+see example - https://checkstyle.org/report_issue.html#How_to_report_a_bug.3F,
+https://github.com/checkstyle/checkstyle/issues/8856,
```

Comment:
please add this link to issue inside https://checkstyle.org/report_issue.html#How_to_report_a_bug.3F page.

Reply:
Done